### PR TITLE
Update Database replication documentation, fix custom(er) constraint

### PR DESCRIPTION
**Pull request by passalaqua**
There was a typo in the database replication section, which was fixed.

I also changed the text slightly.

### CODE REVIEW

File: content/en/docs/appstore/use-content/modules/database-replication.md [hunk 1]
```diff
@@ -261,23 +261,28 @@ In most of the cases, there is no need to change this setting unless you know th
...
-With an import call, you can easily configure both static constraints (meaning, every call with the same value) and dynamic constraints (meaning, based on a value from an attribute).
+Import call is a table mapping associated to a constraint. This constraint may be static, where all calls will be constrained by the same values, or dynamic, where the constraint relies on the value of an attribute.
 
 ### Constraint
 
-In the **Configure import action** dialog box, **Constraints** defines the type of constraint. You can use normal constraints that result in a constraint with all `AND` operators. The constraints will be in alphabetical order according to the constraint number.
+This field defines how all custom constraints will be joined to form the import call constraint:
 
-When selecting **Advanced**, you can set up the constraints yourself using the tokens of the constraint. You can use braces and all the operators supported by the database (`AND`, `OR`, `IN`, `NOT`, etc.). For example: `[&7&] OR ([&1&] AND [&2&])`.
+* **Normal** - All constraints will be joined together with the `AND` operator following their constraint number.
```
Comment (dahfjkg) [inline content/en/docs/appstore/use-content/modules/database-replication.md]: "joined" has connotations, so I would use a different synonym as there is no actual `JOIN` happening. Both on lines 268 and 270.

**Review by dahfjkg — CHANGES_REQUESTED** [review]

File: content/en/docs/appstore/use-content/modules/database-replication.md (hunk 1 above)
Comment (passalaqua) [reply content/en/docs/appstore/use-content/modules/database-replication.md]: Combined?

Comment (dahfjkg) [reply content/en/docs/appstore/use-content/modules/database-replication.md]: Yep, could be "combined"

**Review by dahfjkg — APPROVED** [review]
Passed team review

**Review by MarkvanMents — APPROVED** [review]
Thanks for the PR. Have made some minor changes but looks good.

**Pull request merged**